app/Models/EntryDAO.php

```diff
@@ -129,13 +129,18 @@ protected function autoUpdateDb($errorInfo) {
 						return $this->addColumn($column);
 					}
 				}
-			} elseif ($errorInfo[0] === '42S02' && stripos($errorInfo[2], 'entrytmp') !== false) {	//ER_BAD_TABLE_ERROR
-				return $this->createEntryTempTable();	//v1.7
+			} elseif ($errorInfo[0] === '42S02') {	//ER_BAD_TABLE_ERROR
```

Comment:
I think that the magic value 42S02 should be in class constant.

Reply:
Done in https://github.com/FreshRSS/FreshRSS/pull/2027/commits/036be54ac70b707c1113788a6f9e4146f9981592